exchange/adapter_builders.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/prebid/prebid-server/v3/adapters/adf"
 	"github.com/prebid/prebid-server/v3/adapters/adgeneration"
 	"github.com/prebid/prebid-server/v3/adapters/adhese"
+	"github.com/prebid/prebid-server/v3/adapters/adipolo"
```

Comment:
The test suite job is failing because you didn't add your adapter to the map returned by `newAdapterBuilders` in this file.